frontend/tests/ui/test_contest.py

```diff
@@ -493,6 +502,24 @@ def compare_contestants_list(driver, users_set):
 
     contestants_list = driver.browser.find_elements_by_xpath(
         '//*[@id="ranking"]/div/table/tbody/tr/td[@class="user"]')
-    contestants_set = {item.text for item in contestants_list}
+    # Removing user's name in case they have registered it, in this case all
```

Comment:
No entiendo bien qué estás intentando hacer aquí. puedes darme un ejemplo de lo que ocurre?

Reply:
Ah, ok. todas las pruebas que habíamos hecho sólo guardaban el `username` y `password` de los usuarios que registrabamos, no se guardaba el `name`; este último se muestra en la tabla del `scoreboard` entre paréntesis, tipo:
```
ut_group_w0_1564721415_4:identity_1 (Identity One)
```
Cómo las comparaciones que realiza `different_users` es exacta, y compara `usernames`, tengo que ignorar todo lo que venga después del primer espacio que encuentro en el texto de la celda, ya que ahora estoy incluyendo identidades dentro del `scoreboard` y estas tienen que darse de alta con todos los datos básicos, incluyendo `name`

Reply:
aaaaaah ya veo. qué tal

```suggestion
    # Considering only the username. All unassociated identities are created
    # with a name, which is appended after the username, like:
    # 
    #     ut_group_w0_1564721415_4:identity_1 (Identity One)
```

Reply:
Listo